packages/passwordless-auth-construct/src/otp/otp_challenge_service.test.ts

```diff
@@ -135,130 +57,59 @@ void describe('OTP Challenge', () => {
   });
 
   void describe('createChallenge()', () => {
-    void describe('when SMS', () => {
-      void it('should send and attach sms delivery details', async () => {
-        const smsRequestCreateChallengeEvent = buildCreateAuthChallengeEvent(
-          [],
-          requestOtpSmsMetaData,
-          phoneUserAttributes
-        );
-        const expectedMessageBody = 'Your verification code is';
-        const expectedPhoneNumber = '+15555555555';
-        const expectedOtpLength = 6;
-
-        const createMessageMock = mock.method(
-          mockSmsService,
-          'createMessage',
-          () => expectedMessageBody
-        );
-        const sendMock = mock.method(
-          mockSmsService,
-          'send',
-          // eslint-disable-next-line @typescript-eslint/no-unused-vars
-          (_param: string, _param2: string) => {
-            return;
-          }
-        );
-        const maskMock = mock.method(
-          mockSmsService,
-          'mask',
-          () => expectedPhoneNumber
-        );
-
-        strictEqual(createMessageMock.mock.callCount(), 0);
-        strictEqual(sendMock.mock.callCount(), 0);
-        strictEqual(maskMock.mock.callCount(), 0);
-
-        const result = await otpChallenge.createChallenge(
-          smsRequestCreateChallengeEvent
-        );
-
-        strictEqual(createMessageMock.mock.callCount(), 1);
-        strictEqual(sendMock.mock.callCount(), 1);
-        strictEqual(maskMock.mock.callCount(), 1);
-
-        const actualMessageBody = sendMock.mock.calls[0].arguments[0];
-        const actualPhoneNumber = sendMock.mock.calls[0].arguments[1];
+    void it('should send and attach sms delivery details', async () => {
+      const smsRequestCreateChallengeEvent = buildCreateAuthChallengeEvent(
+        [],
+        requestOtpSmsMetaData,
+        phoneUserAttributes
+      );
+      const expectedPhoneNumber = '+15555555555';
+      const expectedOtpLength = 6;
+
+      const sendMock = mock.method(
+        mockSmsService,
+        'send',
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        (_param: string, _param2: string) => {
```

Comment:
```suggestion
        () => {
```

providing implementation that takes less parameters to fulfill an interface should be legal from compiler pov.

Reply:
In this case they are required because the tests run assertions on the value of the arguments. I added a comment.